src/v2/ion/i18nTransformer.js

```diff
@@ -288,13 +347,22 @@ const isWebAuthnAPIError = ( i18nKey ) => i18nKey.startsWith(WEBAUTHN_API_GENERI
  */
 const getMessage = (message) => {
   if (message.i18n?.key) {
-    const i18nKey = I18N_OVERRIDE_MAPPINGS[message.i18n?.key] ?? message.i18n.key;
+    let i18nKey = message.i18n.key;
+    let i18nParams = message.i18n.params || [];
+
+    if (I18N_OVERRIDE_MAPPINGS[message.i18n?.key]) {
+      i18nKey = I18N_OVERRIDE_MAPPINGS[message.i18n?.key];
+    } else if (I18N_OVERRIDE_WITH_PARAMS_MAP[i18nKey]) {
+      const param = message.i18n.params?.[0];
+      i18nKey = getI8nKeyUsingParams(i18nKey, param);
+      i18nParams = i18nKey.endsWith('custom') ? [param] : [];
```

Comment:
need to test this gets polyfilled in ie

Reply:
`endsWith` does not seem to have any [IE support ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith#browser_compatibility)

Reply:
Do we really have customers who uses IE ? It's 2021 :D

Reply:
It's polyfilled in IE 
<img width="278" alt="Screenshot 2021-05-07 at 11 59 29 AM" src="https://user-images.githubusercontent.com/71431120/117496312-b7f9fb80-af2b-11eb-935b-3a7fc1f50524.png">

Reply:
@pradeepdewda-okta  we have customers who load the widget inside embedded browsers not just IE 😄